eks-anywhere-common/Addons/Partner/SUSE/NeuVector/testJob.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: neuvector-testjob
+  namespace: neuvector
+spec:
+  backoffLimit: 1
+  template:
+    spec:
+      containers:
+        - name: job
+          image: opensuse/leap
+          args:
+            - curl
+            - -ks
+            - --location
+            - --request
+            - POST https://neuvector-svc-controller-api:10443/auth
+            - --header "accept: application/json"
+            - --header "Content-Type: application/json"
```

Comment:
```
  Warning  BuildFailed              4m4s                  kustomize-controller  failed to decode Kubernetes YAML from /tmp/kustomization-1680668155/eks-anywhere-common/Addons/Partner/SUSE/NeuVector/testJob.yaml: MalformedYAMLError: yaml: line 20: did not find expected key
```
Looks the Job has YAML errors please fix those, test it and please resubmit the PR.